x-pack/plugins/security_solution/public/management/hooks/endpoint/use_show_endpoint_response_actions_console.tsx

```diff
@@ -31,6 +31,7 @@ export const useShowEndpointResponseActionsConsole = (): ShowEndpointResponseAct
               commands: getEndpointResponseActionsConsoleCommands(endpointAgentId),
               'data-test-subj': 'endpointResponseActionsConsole',
               prompt: `endpoint-${endpointMetadata.agent.version}`,
+              TitleComponent: () => <>{endpointMetadata.host.name}</>,
```

Comment:
does it make sense to use an `EuiTitle` component here instead of just a jsx fragment?